tests/test_passwords.py

```diff
@@ -0,0 +1,133 @@
+import json
+from urllib import urlencode
+
+import pytest
+from tornado.httpclient import HTTPError
+
+from fixtures import standard_graph, graph, users, groups, session, permissions  # noqa
+from fixtures import fe_app as app  # noqa
```

Comment:
can you add a api test as well to confirm your hashes are making it out correctly on the other side?